olive/model/hf_utils.py

```diff
@@ -75,7 +75,15 @@ class HFModelLoadingArgs(ConfigWithExtraArgs):
             " details for the supported parameters."
         ),
     )
-    # TODO(xiaoyuzhang): seems not working. need to check.
+    # for remote files that require authentication
+    # expose `str` type if needed in the future
+    token: bool = Field(
```

Comment:
double check:
1. token is None: huggingface will not contain any token in the http header for model requests.
2. token is True: huggingface will leverage the token user used in `huggingface-cli login`, if not, raise the error.

One observation: 
for llama2, it is a gated model which need user have the right permission.
1. I cannot load model by `from_pretrained` when I did not login, `token=None` now.
2. I run `huggingface-cli login`, then I can load model by `from_pretrained`, `token=None` now.

Just curious when I login hf, even the token is not True, I still can access.

Reply:
Not block this PR, just provide this information. :)

Reply:
As discussed offline, removed `token` from this PR since it's not needed for local systems. For remote systems, we can handle it later.